tests_integration/__tests__/config-resolution.js

```diff
@@ -50,12 +50,12 @@ test("CLI overrides take precedence", () => {
 
 test("API resolveConfig with no args", () => {
   return prettier.resolveConfig().then(result => {
-    expect(result).toBeNull();
+    expect(result).toEqual({});
   });
 });
 
 test("API resolveConfig.sync with no args", () => {
-  expect(prettier.resolveConfig.sync()).toBeNull();
+  expect(prettier.resolveConfig.sync()).toEqual({});
```

Comment:
Hmm ... this (and the change just above) are breaking changes – is that something we want to avoid?